lib/Dialect/FIRRTL/FIRRTLFolds.cpp

```diff
@@ -1663,11 +1663,14 @@ struct NodeBypass : public mlir::RewritePattern {
     auto node = cast<NodeOp>(op);
     if (node.inner_sym() || !node.annotations().empty())
       return failure();
+
```

Comment:
don't move this here.  Drop it from the pattern.

Reply:
Which incidentally was how I was originally doing it, but nodes weren't being deleted.

Reply:
I think FoldNodeName can delete node with droppable names so I'm not sure there is a problem.